pom.xml

```diff
@@ -10,6 +10,33 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.openjfx</groupId>
+			<artifactId>javafx-base</artifactId>
```

Comment:
you could technically leave out a few of those dependencies - only `javafx-fxml` and `javafx-web` are strictly necessary, not sure whether that's a good idea, though

Reply:
you could also use `git rebase -i` to `drop` that unrelated commit and the one that reverts it, and then do a `git commit --force` (which is fine to use for PRs)